### PR TITLE
Add projector.add_to_preview and tests.

### PR DESCRIPTION
**Pull request by reiterl**

### CODE REVIEW

File: openslides_backend/action/actions/projector/add_to_preview.py [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+    """
+    Action to projector project.
+    """
+
+    model = Projector()
+    schema = DefaultSchema(Projection()).get_default_schema(
+        required_properties=["content_object_id"],
+        optional_properties=["options", "stable", "type"],
+        additional_required_fields={
+            "ids": {"type": "array", "items": required_id_schema}
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/projector/add_to_preview.py:23]: The ids should also be unique.

File: openslides_backend/action/actions/projector/add_to_preview.py [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    def get_updated_instances(self, action_data: ActionData) -> ActionData:
+        for instance in action_data:
+            # check content object and get meeting id.
+            content_object = self.datastore.get(
+                string_to_fqid(instance["content_object_id"]), ["meeting_id"]
+            )
+            meeting_id = content_object["meeting_id"]
+
+            # add the preview projections
+            for projector_id in instance["ids"]:
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/projector/add_to_preview.py:37]: is it checked, that the projector ids belong to the meeting? Maybe add a test for this.

File: openslides_backend/action/actions/projector/add_to_preview.py [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+                max_weight = self.get_max_projection_weight(projector_id)
+                data = {
+                    "meeting_id": meeting_id,
+                    "preview_projector_id": projector_id,
+                    "weight": max_weight + 1,
+                }
+                for field in ("content_object_id", "options", "stable", "type"):
```
Comment (FinnStutzenstein) [inline openslides_backend/action/actions/projector/add_to_preview.py:44]: `content_object_id` is required, you can add it directly to `data`.

**Review by FinnStutzenstein — CHANGES_REQUESTED** [review]
Looks good so far

File: openslides_backend/action/actions/projector/add_to_preview.py (hunk 1 above)
Comment (reiterl) [reply openslides_backend/action/actions/projector/add_to_preview.py:23]: Okay, done.

File: openslides_backend/action/actions/projector/add_to_preview.py (hunk 3 above)
Comment (reiterl) [reply openslides_backend/action/actions/projector/add_to_preview.py:44]: Done.

File: openslides_backend/action/actions/projector/add_to_preview.py (hunk 2 above)
Comment (reiterl) [reply openslides_backend/action/actions/projector/add_to_preview.py:37]: Yes, it is indirectly checked. Added a test for this. You cannot add a projection where meeting_id differ with content_object meeting_id.

**Review by FinnStutzenstein — APPROVED** [review]

**Review by jsangmeister — APPROVED** [review]

**Pull request merged**